src/test/assembly/x86_64-sse_crc.rs

```diff
@@ -0,0 +1,12 @@
+// assembly-output: emit-asm
+// compile-flags: --target=x86_64-unknown-linux-gnu --crate-type staticlib -Ctarget-feature=+sse4.2
+// needs-llvm-components: x86
```

Comment:
If you want this to work, you will need to either:

1. Make this a `#[no_core]` test (allowing this test to run on any architecture);
2. Make this a `// only-x86` test instead (removing the `--target` flag and the `needs-llvm-components` annotation).

Reply:
Done.